Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocAutoCompleteReferenceValueControlObject.cs

```diff
@@ -270,7 +270,7 @@ protected override ElementScope GetLabeledElementScope ()
       if (IsReadOnly())
         return Scope.FindChild ("Command");
 
-      if (Scope.Browser.IsInternetExplorer())
+      if (Scope.Browser == Browser.InternetExplorer)

```

Comment:
I'm thinking we might want to keep the extension method for Coypu.Browser, just not public (so, copied in web and Objectbinding.web). this way, we have a more obvious way of finding usages?